src/main/kotlin/racingcar/domain/car/Car.kt

```diff
@@ -0,0 +1,16 @@
+package racingcar.domain.car
+
+import racingcar.domain.car.action.MovingStrategy
+import racingcar.domain.car.status.Name
+import racingcar.domain.car.status.Position
+
+data class Car(private val carName: String) {
```

Comment:
```suggestion
data class Car(val name: Name, position: Position = Position(0)) {
var position: Position = position
private set 
```

carName과 같이 불필요한 프로퍼티를 가지고 있기 보단 위처럼 구현하는것도 좋은 방법이 될것 같습니다. 
String Type의 carName을 받을수 있는 secondary constructor 를 선언해서 구현할수 있을거에요 😄

Reply:
네, `Car` 클래스의 생성자를 유연하게 사용할 수 있도록 secondary constructor 를 추가하였습니다! 😃 

- https://github.com/next-step/kotlin-racingcar/commit/bcc989332e525313c974736dd1c95785b0245b18